lib/teleterm/config.go

```diff
@@ -31,6 +30,8 @@ type Config struct {
 	ShutdownSignals []os.Signal
 	// HomeDir is the directory to store cluster profiles
 	HomeDir string
+	// CertsDir is the directory where the certs are stored
```

Comment:
It's not clear in the comment _what_ certs you are referring to. Can you clarify by mentioning these are the certs used to authenticate with `tsh`?

Reply:
Done, are they better now?